eng/common/scripts/Invoke-DevOpsAPI.ps1

```diff
@@ -0,0 +1,42 @@
+. "${PSScriptRoot}\logging.ps1"
+. "${PSScriptRoot}\Rest-Methods.ps1"
+
+$DevOpsAPIBaseURI = "https://dev.azure.com/{0}/{1}/_apis/{2}/{3}?api-version=6.0"
+
+function Get-RestMethodHeaders ($token) {
```

Comment:
For DevOps if I recall it requires that the token is base64 encoded we should probably add that information in here somehow. Maybe in a comment or perhaps the variable name.